atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/Cell.java

```diff
@@ -66,9 +67,11 @@ private void validateNameValid(byte[] name) {
                     "name must be no longer than {}.",
                     MAX_NAME_LENGTH);
         } catch (IllegalArgumentException e) {
-            log.debug("Cell creation that was attempted was: {}; since the vast majority of people encountering this "
+            log.error("Cell creation that was attempted was: {}; since the vast majority of people encountering this "
```

Comment:
This error message is pretty non-descriptive... can we include the exception message above? Name exceeded maximum name length